lib/livebook_web/live/user_component.ex

```diff
@@ -35,7 +35,12 @@ defmodule LivebookWeb.UserComponent do
         phx-hook="UserForm"
       >
         <div class="flex flex-col space-y-5">
-          <.text_field field={f[:name]} label="Display name" spellcheck="false" />
+          <.text_field
+            field={f[:name]}
+            label="Display name"
+            spellcheck="false"
+            readonly={readonly?()}
```

Comment:
It may be surprising that they can't edit, we can give more indication with disabled:

```suggestion
            disabled={readonly?()}
```